### PR TITLE
Added content and fixed typos to markdown file

### PR DESCRIPTION
**Pull request by veemcb**

### CODE REVIEW

File: How_the_toolkit_works.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ How the toolkit works
...
+The toolkit content is stored on the github servers under the <a href="https://github.com/astro4dev/OAD-Data-Science-Toolkit" target="_blank">OAD-Data-Science-Toolkit</a> repository. Content is added by individual contributors by cloning the repository, making the changes, then creating a pull request. A description of the workflow can be found <a href="https://github.com/astro4dev/OAD-Data-Science-Toolkit/blob/master/How_to_upload_content.md" target="_blank">here</a>. Github members of the astro4dev data science working group are automatically informed about the new pull request. The request is then verified before it is added to the main astro4dev repository.
```
Comment (brucellino) [inline How_the_toolkit_works.md]: Hi @veemcb  (can I still call you V ?)
I would suggest that each sentence is put on a new line, else git thinks the entire paragraph needs to be changed. This makes it very difficult to catch little errors and leads to churn in the code.

File: How_the_toolkit_works.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ How the toolkit works
...
 
 ## Step 2 - Adding the content to the database
 
-Once the content has been uploaded to GitHub it has to be added to the MySQL database. This is done by a member of the astro4dev data science working group. The database is composed of many tables which can be dividied into two types.
+Once the content has been uploaded to GitHub it has to be added to the MySQL database. This is done by a member of the astro4dev data science working group. The database is composed of many tables which can be divided into two types.
```
Comment (brucellino) [inline How_the_toolkit_works.md:12]: Agree.

File: How_the_toolkit_works.md [hunk 3]
```diff
@@ -19,4 +19,8 @@ The content tables have one word names, e.g. _skills_ and contain information su
 ### The connector tables
 These tabels are required to connect the content tables in the database. Connector tables all contain a double underscore '__' so they can easily be identified. They have been created so that new tables can easily be added later. Also, it drastically cuts down the size of each individual table (look up _relational databases_). If we wish to connect two content tables, such as _skills_ and _examples_ we have to create an entry in the connector table _skills__examples_. Each time a new piece of content is added to GitHub a *minimum of three* tables have to be updated. Say an example has been added. For it to be properly added to the database, each table with _"example"_ in it's name has to be updated. Here is an example of how the tables are connected.
 
-<p align="center"><img src="https://raw.githubusercontent.com/astro4dev/OAD-Data-Science-Toolkit/master/img/database_structure.png" alt="database structure"/></p>
\ No newline at end of file
+<p align="center"><img src="https://raw.githubusercontent.com/astro4dev/OAD-Data-Science-Toolkit/master/img/database_structure.png" alt="database structure"/></p>
```
Comment (brucellino) [inline How_the_toolkit_works.md]: Agree

**Review by brucellino — APPROVED** [review]
Basically agree with these changes. Just a small suggestion to keep prose contributions separated by sentence.

**Review by paultheastronomer — APPROVED** [review]
I made sure our changes did not conflict.

**Pull request merged**